src/code/z_bgcheck.c

```diff
@@ -379,31 +380,41 @@ s32 CollisionPoly_CheckZIntersectApprox(CollisionPoly* poly, Vec3s* vtxList, f32
                                             y, zIntersect);
 }
 
-#ifdef NON_MATCHING
 s32 CollisionPoly_LineVsPoly(BgLineVsPolyTest* a0) {
     static Vec3f polyVerts[3]; // D_801EDB70
     static Plane plane;        // D_801EDB98
```

Comment:
We don't really have a good way to deal with in-function static bss reordering, which will be a problem in the future, specially with the big amount of bss this file has.
My recommendation is to leave this function as NON_MATCHING with a note saying this function matches but bss

Reply:
Should the same be done with `SphVsPoly`, which already has its static bss in-functioned?

Reply:
> Should the same be done with `SphVsPoly`, which already has its static bss in-functioned?

I think so. I'm kind of surprised it hasn't been troublesome yet.
Note I've usually left the comment `// needs in-function static bss` for functions that match but just need in function bss. That way it is easier to find them later.

Reply:
Bump on engineer's comment, for these named bss variables, can you prefix `s`